src/notation/internal/notationinteraction.cpp

```diff
@@ -1937,18 +1937,26 @@ void NotationInteraction::startEditText(Element* element, const PointF& cursorPo
 
 void NotationInteraction::editText(QKeyEvent* event)
 {
-    IF_ASSERT_FAILED(m_textEditData.element) {
+    if (!m_textEditData.element && selection()->element()) {
```

Comment:
should probably rename to just m_editData now we're using it for other types of elements too